### PR TITLE
cinder-csi-plugin: upgrade resizer and attacher side car container

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

csi-resizer 0.5.1=>1.1.0
csi-attacher 2.2.1=>3.1.0
**Which issue this PR fixes(if applicable)**:
fixes #

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jichenjc** [comment]
duplicate to https://github.com/kubernetes/cloud-provider-openstack/pull/1314

**Comment by jichenjc** [comment]
@ramineni @lingxiankong please help to comment, thanks

File: manifests/cinder-csi-plugin/cinder-csi-controllerplugin-rbac.yaml [hunk 1]
```diff
@@ -26,6 +26,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["csinodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments/status"]
+    verbs: ["patch"]
```
Comment (ramineni) [inline manifests/cinder-csi-plugin/cinder-csi-controllerplugin-rbac.yaml:31]: could we remove nodes at L21 , its not required anymore I suppose.

**Review by ramineni — CHANGES_REQUESTED** [review]
@jichenjc Thanks for updating. Few comments

1. please make sure rbac is in consistent with the rbac of respective repos of that specified release. For example, I see there is no "update"  verbs required for some resources.
2. We might have to set 'handle-volume-inuse-error' flag to false, as we support online expansion of volumes.

File: manifests/cinder-csi-plugin/cinder-csi-controllerplugin-rbac.yaml (hunk 1 above)
Comment (jichenjc) [reply manifests/cinder-csi-plugin/cinder-csi-controllerplugin-rbac.yaml:31]: yes, looks like nodes removed from here
https://github.com/kubernetes-csi/external-attacher/blob/master/deploy/kubernetes/rbac.yaml

**Comment by jichenjc** [comment]
@ramineni thanks for the review, updated accordingly

**Comment by ramineni** [comment]
Thanks. Looks good. waiting for tests pass .

**Comment by jichenjc** [comment]
@ramineni @lingxiankong can you help a review on this?thanks

File: charts/cinder-csi-plugin/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: latest
 description: Cinder CSI Chart for OpenStack
 name: openstack-cinder-csi
-version: 1.3.1
+version: 1.3.2
```
Comment (ramineni) [inline charts/cinder-csi-plugin/Chart.yaml:5]: version in conflict , pls update

Comment (jichenjc) [reply charts/cinder-csi-plugin/Chart.yaml]: updated, thx

**Comment by lingxiankong** [comment]
Just curious, are there any tests related to resize (volume expansion)?

**Comment by ramineni** [comment]
> Just curious, are there any tests related to resize (volume expansion)?
> 
> /lgtm

@lingxiankong I did enable the tests before , but had to revert back as citynetwork cloud doesnt support it . Do you have any idea ,if its still the case ?  https://github.com/kubernetes/cloud-provider-openstack/pull/975

**Comment by lingxiankong** [comment]
> > Just curious, are there any tests related to resize (volume expansion)?
> > /lgtm
> 
> @lingxiankong I did enable the tests before , but had to revert back as citynetwork cloud doesnt support it . Do you have any idea ,if its still the case ? #975

Let's see the result of https://github.com/kubernetes/cloud-provider-openstack/pull/1406

**Comment by ramineni** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**